doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1222,6 +1222,33 @@ Zigbee
 
 The issues in this section are related to the :ref:`ug_zigbee` protocol.
 
+.. rst-class:: v2-9-0 v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
+
+NCSIDB-1411: Clearing config data in response to the Leave Network command is not fully performed
+   Configuration data of ZCL Reporting feature is not cleared when processing the "Leave Network" command, which means that compliance with the specification is incomplete.
+
+   The Zigbee BDB 3.0 Specification, section 9.4 says that all Zigbee persistent data (with exceptions ..) must be cleared in response to the Mgmt_Leave_req command, however the stack leaves the ZCL Reporting parameter values ​​set (cached in memory) and continues to use them.
+
+   **Workaround:** You can supplement the processing of the "Leave Network" command with ZCL Reporting parameter clearing by adding a :c:func:`zb_zcl_init_reporting_info` call in the ``ZB_ZDO_SIGNAL_LEAVE`` handler.
```

Comment:
```suggestion
   **Workaround:** You can supplement the processing of the leave network command with ZCL Reporting parameter clearing by adding a :c:func:`zb_zcl_init_reporting_info` call in the ``ZB_ZDO_SIGNAL_LEAVE`` handler.
```

Reply:
Same proposal as above - Leave Network command by starting the name with capital letter

```suggestion
   **Workaround:** You can supplement the processing of the Leave Network command with ZCL Reporting parameter clearing by adding a :c:func:`zb_zcl_init_reporting_info` call in the ``ZB_ZDO_SIGNAL_LEAVE`` handler.
```

@annwoj  please comment

Reply:
I'm fine with that approach